libmei/addons/att.h

```diff
@@ -42,13 +42,13 @@ class Att : public AttConverterBase {
     static data_STAFFREL_basic StaffrelToStaffrelBasic(data_STAFFREL staffrel);
 
     /** Dummy string converter */
-    std::string StrToStr(std::string str) const;
+    std::string StrToStr(const std::string str) const;
 
     /** @name Basic converters for writing */
     ///@{
-    std::string DblToStr(double data) const;
-    std::string IntToStr(int data) const;
-    std::string VUToStr(data_VU data) const;
+    std::string DblToStr(const double data) const;
```

Comment:
What difference does it make?

Reply:
Basically none, except for code consistency.

Reply:
Consistency with what?